java-client/src/main/java/org/opensearch/client/transport/aws/AwsSdk2Transport.java

```diff
@@ -8,12 +8,18 @@
 
 package org.opensearch.client.transport.aws;
 
+import io.netty.channel.ChannelPipelineException;
```

Comment:
I think we should not catch anything specific to `netty` or alike: AFAIK the client transport is pluggable in AWS SDK, we cannot cover all possible client impl

Reply:
Ok. This means that Netty exceptions will all be rethrown wrapped in IOException. I can revert this.

Reply:
Thank you